fft/src/errors.rs

```diff
@@ -16,3 +16,14 @@ pub enum FFTError {
     #[error("Couldn't create a new Metal compute pipeline: {0}")]
     MetalPipelineError(String),
 }
+
+impl From<FieldError> for FFTError {
+    fn from(error: FieldError) -> Self {
+        match error {
+            FieldError::DivisionByZero => {
+                panic!("Can't divide by zero during FFT");
```

Comment:
Please try not to panic in library crates. You should always give our downstream users the opportunity to handle failure.